src/Entity/Styles/README.md

```diff
@@ -0,0 +1,4 @@
+# CSL Styles
+
+These styles are pulled from https://github.com/citation-style-language/styles with customized formatting to more
+accurately match the defined styles.
```

Comment:
PR for that project or fork it for ourselves?

Reply:
The styles are hosted locally in this PR. there were a fair amount of changes and especially with the linking title.